### ISSUE TITLE
What happens if we use a larger portfolio?

### ISSUE THREAD

**Issue by cmaes** [EmilienDupont/markowitz#7]
It would be interesting to solve this problem for a larger candidate portfolio set. Say every stock in the S&P 500. It may not be solvable within 10s. But we might see better returns.

**Comment by EmilienDupont** [comment]
Yes, we could try that. In that case, should we load all the data as we do now? Or precompute a covariance matrix etc...
